pyproject.toml

```diff
@@ -70,4 +70,3 @@ warn_unused_ignores = false
 [tool.pytest.ini_options]
 minversion = "6.0"
 log_cli_level = "INFO"
-asyncio_mode = "auto"
```

Comment:
This one is necessary for itests to work.

Reply:
itests? Do you know why pytest issues a warning about it in that case?

Reply:
itest = integration test.

I recall some warning in the past but a recent run seems clear:
https://github.com/canonical/loki-k8s-operator/actions/runs/3940115006/jobs/6740823919#step:5:334

Reply:
Unless literally every single test uses `asyncio_mode`, the warning is nag.

Reply:
Thanks. Hmm, but that link is without the removal of `asyncio_mode = "auto"`. Reinstated.